File: 1550-three-consecutive-odds/1550-three-consecutive-odds.cpp
class Solution {
public:
    bool threeConsecutiveOdds(vector<int>& arr) {
        int consecutive_odd = 0;

        for (int i = 0; i < arr.size(); i++){
            if (arr[i] % 2 != 0){
                consecutive_odd ++;
            }
            else{
                consecutive_odd = 0;
            }
            if (consecutive_odd == 3){
                return true;
            }
        }
        return false;
    }
};

File: 1550-three-consecutive-odds/1550-three-consecutive-odds.py
class Solution:
    def threeConsecutiveOdds(self, arr: List[int]) -> bool:
        consecutive_odd = 0
        for i in range(len(arr)):
            if arr[i] % 2 != 0:
                consecutive_odd += 1
            else:
                consecutive_odd = 0
            
            if consecutive_odd == 3:
                return True
        return False
            
        